src/Twilio/Http/Net35/WebRequestWrappers.cs

```diff
@@ -0,0 +1,152 @@
+﻿#if NET35
+using System;
+using System.Collections.Specialized;
+using System.IO;
+using System.Net;
+
+namespace Twilio.Http.Net35
+{
+    /// <summary>
+    /// These classes exist to wrap MSFT's WebRequest interfaces to enable unit testing of 
+    /// WebRequestClient.
+    /// </summary>
+    
+    public class HttpWebRequestFactory
+    {
+        public virtual IHttpWebRequestWrapper Create(Uri uri)

```

Comment:
ℹ️  virtual means the method can be overriden which is required for creating a mock of it.